src/ng-select/config.ts

```diff
@@ -0,0 +1,19 @@
+import { InjectionToken } from '@angular/core';
+import { NgSelectConfig } from './ng-select.types';
+
+export const ngSelectConfigDefaults = {
```

Comment:
It would be better to create injectable config class as ng-bootstrap does https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/datepicker/datepicker-config.ts

Reply:
Changed config to service. But you must note that unlike my previous solution this approach creates breaking changes for everyone who used custom configuration before